src/Str.php

```diff
@@ -63,12 +72,16 @@ public static function symmetricSplit($subject, $delimiter, $limit, $default = n
      *
      * @param string $subject
      * @param string $delimiter
-     * @param int    $limit
+     * @param ?int $limit
```

Comment:
`?int` is not equal to `int $bla = null`. You can't pass null explicitly with the latter. Remove the `?` again.